tests/plugins/test_plugins_registry.py

```diff
@@ -107,3 +113,19 @@ def test_tpu_spawn_debug_plugins_registry(tmpdir):
     trainer = Trainer(plugins=plugin)
 
     assert isinstance(trainer.training_type_plugin, TPUSpawnPlugin)
+
+
+def test_ddp_spawn_training_type_plugins_registry(tmpdir):
```

Comment:
we could simply parameterize this test with the name + corresponding expected plugin, e.g, these cases:

```python
("ddp_find_unused_parameters_false", DDPPlugin)
("ddp_spawn_find_unused_parameters_false", DDPSpawnPlugin)
("deepspeed_find_unused_parameters_false", DeepspeedPlugin)
```

and sharded etc. 
What do you think?

Reply:
Ofcourse.

Reply:
Updated!